authorization/invitation/service/invitation_service.go

```diff
@@ -157,9 +157,9 @@ func (s *invitationServiceImpl) Issue(ctx context.Context, issuingUserId uuid.UU
 				inv.ResourceID = &inviteToResource.ResourceID
 			}
 
-			error := s.Repositories().InvitationRepository().Create(ctx, inv)
-			if error != nil {
-				return errors.NewInternalError(ctx, error)
+			err = s.Repositories().InvitationRepository().Create(ctx, inv)
+			if err != nil {
+				return errors.NewInternalError(ctx, err)
```

Comment:
I don't thin that we should wrap the underlying repository error into an `InternalError` here. In worst case, it'll be done at the API level anyways

Reply:
sure, let me change that